src/azure-cli/azure/cli/command_modules/container/tests/latest/test_container_commands.py

```diff
@@ -243,6 +243,41 @@ def test_container_create_with_msi_scope(self, resource_group, resource_group_lo
                          self.exists('ipAddress.ip'),
                          self.check('containers[0].image', '{image}')])
 
+
+    # Test create container using auto generated domain name label scope.
+    @live_only()
```

Comment:
Why does this test not support recording mode?

Reply:
How do we make it support recording mode?

Reply:
@angouloure You'll just have to remove `@live_only()`. Record is the default behavior.

The test documentation is here: https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md

I can walk you through our tests if you have any issues.